daemon/algod/api/algod.oas2.json

```diff
@@ -4142,9 +4142,8 @@
           }
         },
         "approval-program-hash": {
-          "description": "Hash digest of the approval program executed in transaction.",
-          "type": "string",
-          "format": "bytes"
+          "description": "Base32 encoded SHA512_256 hash digest of the approval program executed in transaction.",
```

Comment:
I vote for don't base32 encode. It will be confusing because it'll look very similar to the logicsig address, but it's not for two reasons: no `Program` prefix, and no checksum. 32-byte binary string seems fine to me

Reply:
alright, seems okay with me